src/components/interactive-map.js

```diff
@@ -116,12 +184,19 @@ export default class InteractiveMap extends PureComponent {
       overflow: 'hidden'
     };
 
+    const eventCanvasStyle = Object.assign({}, {
```

Comment:
what's the reason for `Object.assign` here? You're not merging any existing objects right?